### DISCUSSION TITLE
Map

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1989] [Q&A] [answered]
Hello

The map is not working.

Were changes made?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz Nothing should have changed in your instance unless you upgraded to a new version. If you could post a link to the map I might be able to see what's going on.

**Reply by SandraEliz** [reply]
https://prep-odsmexico.github.io/site/3-1-1/

**Comment by brockfanning** [comment, marked as answer]
@SandraEliz That issue has been fixed in a "hotfix" release of Open SDG. In this line you are using version 2.1.0 of Open SDG: https://github.com/prep-odsmexico/site/blob/develop/_config.yml#L4

You could change that line to point to `2.1.1` instead, as a quick fix.

Alternatively, you could upgrade to 2.2.0, which is now the most current version. If you take this approach, you would need to follow the upgrade instructions for 2.2.0: https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-2-2-0/
